src/Main.fs

```diff
@@ -41,8 +41,18 @@ let parseFSharpProject (com: ICompiler) (checker: FSharpChecker) (projCode: stri
         | ".fsx" ->
             let otherFlags =
                 com.Options.symbols |> Array.map (sprintf "--define:%s")
+            let comOptionsprojFile =
+              try
+                let test = Path.GetDirectoryName( com.Options.projFile )
```

Comment:
See [the exception detail](https://gist.github.com/davidpodhola/44c7a1bfbbc92a0da708) why `Path.GetDirectoryName` is used